src/index.html

```diff
@@ -1,22 +1,463 @@
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/portable-speakers-b.png"
+                      alt="Portable Speakers photo B"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Portable speakers</h3>
+              </article>
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/earphones-photo-a.png"
+                      alt="Earphones photo A"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/earphones-photo-b.png"
+                      alt="Earphones photo B"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Earphones</h3>
+              </article>
+              <article class="category">
+                <div class="category__photos">
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/smart-home-A.png"
+                      alt="Smart Home photo A"
+                      class="category__photo"
+                    />
+                  </a>
+                  <a
+                    href="#"
+                    class="category__link"
+                  >
+                    <img
+                      src="./images/smart-home-b.png"
+                      alt="Smart Home photo B"
+                      class="category__photo"
+                    />
+                  </a>
+                </div>
+
+                <h3 class="category__title">Smart Home</h3>
+              </article>
+
+              <div class="category__button">
+                <button class="button">
+                  <a
+                    href="#"
+                    class="button__link"
+                  >
+                    View all
+                  </a>
+                </button>
+              </div>
+            </div>
+          </section>
+          <section
+            class="about-us"
+            id="about-us"
+          >
+            <div class="photo-news">
+              <img
+                src="./images/about-us.png"
+                alt="news"
+                class="photo-news__img"
+              />
+            </div>
+
+            <div class="about-us__news">
```

Comment:
the `about` section has the same background as the `contact-us`, this isn't a explicit requirement on the checklist but it gives a weird appearance on the website.

![image](https://github.com/user-attachments/assets/c78b0cc0-8389-45d1-ba7a-5cb2b670ed7b)


on Figma this section looks like this:

![image](https://github.com/user-attachments/assets/afe8694d-2778-47d6-ae00-2700146b65a3)